tests/StorageApiTestCase.php

```diff
@@ -43,6 +45,16 @@ abstract class StorageApiTestCase extends ClientTestCase
     /** @var Tokens */
     protected $tokens;
 
+    /**
+     * @var string
+     */
+    private $tokenId;
+
+    /**
+     * @var string
+     */
+    private $lastEventId;
```

Comment:
IMHO jsi to zapomněl odebrat v tom \Keboola\Test\Common\ComponentsEventsTest::$lastEventId